### PR TITLE
add sdk python demo

### PR DESCRIPTION
**Pull request by lvhan028**
Thanks for your contribution and we appreciate it a lot. The following instructions would make your pull request more healthy and more easily receiving feedbacks. If you do not understand some items, don't worry, just make the pull request and seek help from maintainers.

## Motivation

Add python demo, showing how to use SDK Python API

## Modification

demo/python/*

### CODE REVIEW

**Review by lzhangzz — APPROVED** [review]

File: demo/python/image_classification.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import argparse
+
+import cv2
+from mmdeploy_python import Classifier
```
Comment (grimoire) [inline demo/python/image_classification.py:5]: `mmdeploy_python` will not be added to the python environment when building from the source.

File: mmdeploy/codebase/mmseg/deploy/segmentation_model.py [hunk 2]
```diff
@@ -106,7 +106,7 @@ def forward_test(self, imgs: torch.Tensor, *args, **kwargs) -> \
         """
         outputs = self.wrapper({self.input_name: imgs})
         outputs = self.wrapper.output_to_list(outputs)
-        outputs = [out.detach().cpu().numpy() for out in outputs]
+        outputs = [out.detach().cpu().numpy() for _, out in outputs]
```
Comment (grimoire) [inline mmdeploy/codebase/mmseg/deploy/segmentation_model.py:109]: visualize  failed when exporting the model.

File: demo/python/object_detection.py [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+    indices = [i for i in range(len(bboxes))]
+    for index, bbox, label_id in zip(indices, bboxes, labels):
+        [left, top, right, bottom], score = bbox[0:4].astype(int), bbox[4]
+        if score < 0.3:
+            continue
+
+        cv2.rectangle(img, (left, top), (right, bottom), (0, 255, 0))
+
+        if masks:
```
Comment (grimoire) [inline demo/python/object_detection.py:35]: Mask is a list of empty ndarray, even when model is retinanet.

Comment (lvhan028) [reply demo/python/object_detection.py:35]: `mask` is only for instance seg.

Comment (grimoire) [reply demo/python/object_detection.py:35]: object_detection failed because retinanet will also enter this branch.

File: demo/python/image_classification.py (hunk 1 above)
Comment (lvhan028) [reply demo/python/image_classification.py:5]: PYTHONPATH should be set. I am going to note it down in PR #531

File: mmdeploy/codebase/mmseg/deploy/segmentation_model.py (hunk 2 above)
Comment (lvhan028) [reply mmdeploy/codebase/mmseg/deploy/segmentation_model.py]: fixed.

**Review by grimoire — APPROVED** [review]

**Review by lzhangzz — APPROVED** [review]
